hyperbahn/client.go

```diff
@@ -58,6 +58,24 @@ const (
 
 const hyperbahnServiceName = "hyperbahn"
 
+func (f *FailStrategy) UnmarshalYAML(unmarshal func(v interface{}) error) error {
+	var strategy string
+	if err := unmarshal(&strategy); err != nil {
+		return err
+	}
+	var err error
```

Comment:
nit: instead of creating an err variable, you should just return in the default case
```go
switch strategy {
  case "fatal":
    *f = FailStrategyFatal
  case "ignore":
    *f = FailStrategyIgnore
  default:
    return fmt.Errorf(...)
}
return nil
```